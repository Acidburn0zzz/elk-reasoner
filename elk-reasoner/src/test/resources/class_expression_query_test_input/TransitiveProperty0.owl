Prefix(:=<owlapi:ontology#ont31#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<owlapi:ontology#ont31>

Declaration(Class(<http://example.org/A>))
Declaration(Class(<http://example.org/C>))
Declaration(ObjectProperty(<http://example.org/T>))
############################
#   Object Properties
############################

# Object Property: <http://example.org/T> (<http://example.org/T>)

TransitiveObjectProperty(<http://example.org/T>)


############################
#   Classes
############################

# Class: <http://example.org/A> (<http://example.org/A>)

SubClassOf(<http://example.org/A> ObjectSomeValuesFrom(<http://example.org/T> ObjectSomeValuesFrom(<http://example.org/T> <http://example.org/C>)))


)