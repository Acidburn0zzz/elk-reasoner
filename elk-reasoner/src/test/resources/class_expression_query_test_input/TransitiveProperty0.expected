Prefix(:=<owlapi:ontology#ont32#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<owlapi:ontology#ont32>

Declaration(Class(<http://example.org/A>))
Declaration(Class(<http://example.org/C>))
Declaration(ObjectProperty(<http://example.org/T>))

############################
#   Classes
############################

# Class: <http://example.org/A> (<http://example.org/A>)

SubClassOf(<http://example.org/A> ObjectSomeValuesFrom(<http://example.org/T> <http://example.org/C>))



SubClassOf(ObjectSomeValuesFrom(<http://example.org/T> <http://example.org/C>) owl:Thing)
)